frontend/src/modules/machinery/components/CaighdeanTranslation.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import CaighdeanTranslation from './CaighdeanTranslation';
+
+
+describe('<CaighdeanTranslation>', () => {
+    it('renders the component properly', () => {
+        const source = {
```

Comment:
You could move this to a constant outside of the `describe` block, called `SOURCE`, and use it just like you did here. Note that you'd not pass anything to the component, since it doesn't actually require any props.